content/pt-br/blog/_posts/2023-03-10-image-registry-change.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: blog
+title: "Redirecionamento do registry de imagens k8s.gcr.io para registry.k8s.io - O que você precisa saber"
+date: 2023-03-10T17:00:00.000Z
+slug: image-registry-redirect
+---
+
+**Autores**: Bob Killen (Google), Davanum Srinivas (AWS), Chris Short (AWS), Frederico Muñoz (SAS
+Institute), Tim Bannister (The Scale Factory), Ricky Sadowski (AWS), Grace Nguyen (Expo), Mahamed
+Ali (Rackspace Technology), Mars Toktonaliev (independent), Laura Santamaria (Dell), Kat Cosgrove
+(Dell)
+
+
+Na segunda-feira, dia 20 de Março, o registry de imagens k8s.gcr.io [vai ser redirecionado para o registry da comunidade](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/),**registry.k8s.io**.
+
+
+## TL;DR: O que você precisa saber sobre essa mudança
```

Comment:
“Sumário” instead of “TL;DR”, because it seems clearer.

Reply:
Instead of using "Sumário" I can just remove it - what do you think?

Reply:
ok